src/main/java/com/sixt/service/framework/servicetest/service/ServiceUnderTestImpl.java

```
@@ -146,6 +146,17 @@ public String sendHttpPost(String path, String data) throws Exception {
     }
 
     @Override
+    public <TYPE extends Message> List<TYPE> getEventsOfType(Class<TYPE> eventClass) {
+
+        if (eventHandler == null) {
+            logger.warn("Event handler has not been initialized. Create the " +
```

this should probably throw an exception instead of just a warning